tinker/events/events_controller.py

```diff
@@ -472,17 +476,17 @@ def event_dates_in_date_range(self, list_of_dates, start, end):
             if start != 0:
                 A = self.compare_datetimes(start, event_start) <= 0  # Search start is before event start
```

Comment:
I really don't like that we are using letters as variable names here. I know it wasn't your code, but could you update those with real names?